cypress/integration/(Regression).js

```diff
@@ -3,4 +3,32 @@
  * Licensed under the MIT License.
```

Comment:
The file above says `--spec "cypress/integration/(Regression).spec.js"` which I thought intended to refer to this file but I'm not seeing the rename.

I was expecting this file to also be renamed with the `.spec.js` to the command above can find it.

Reply:
It even says in your test run `Can't run because no spec files were found.` ...

Reply:
Good catch. An oversight.